.github/workflows/nightly.yaml

```
@@ -0,0 +1,36 @@
+name: Nightly builds
+on:
+  schedule:
```

```suggestion
  workflow_dispatch: # Allows manual trigger
  push:
    branches:
    - master
```

I feel like triggering every night is a bit too much, and it spams the Actions page

---

Look here how the [actions](https://github.com/zyedidia/micro/actions) log is already "spammed" by every automated "[Build and Test](https://github.com/zyedidia/micro/actions/workflows/test.yaml)" trigger for approved (for building) PR.
In my opinion one build per day more doesn't make the difference here.

Wikipedia supports my understanding of what a [nightly](https://en.wikipedia.org/wiki/Daily_build) is used for. :wink: 

Does a nightly really need to be started manually? We should keep it simple and trigger it once a day for the actual present `HEAD` of the `master`.

The `nightly` tag could become handy in the moment we like to go the way to create a kind of nightly "release" to publish it globally (no GitHub account needed for the download). But then the `nightly` tag must be set automatically and a third party action must be used for the release handling.